pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -165,12 +164,32 @@ func parseCRILog(log []byte, msg *logMessage) error {
 	return nil
 }
 
+// jsonLog is a log message, typically a single entry from a given log stream.
+type jsonLog struct {
+	// Log is the log message
+	Log string `json:"log,omitempty"`
+	// Stream is the log source
+	Stream string `json:"stream,omitempty"`
+	// Created is the created timestamp of log
+	Created time.Time `json:"time"`
+	// Attrs is the list of extra attributes provided by the user
+	Attrs map[string]string `json:"attrs,omitempty"`
```

Comment:
attrs is unused, drop it so we don't incur deserialization cost

Reply:
good point